ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/util/Json.java

```diff
@@ -167,7 +160,7 @@ public static ObjectNode readObjectNode(byte[] data) throws IOException {
     // Read data into a tree
     JsonNode root = MAPPER.readTree(data);
     // Check that we have an object, because treeToValue won't
-    if (root == null || !root.isObject()) {
+    if (!root.isObject()) {
```

Comment:
starting with jackson-databind 2.10.0 `ObjectMapper.readTree` will return `MissingNode` where it once returned `null`